filebeat/tests/system/test_registrar.py

```diff
@@ -975,8 +975,8 @@ def test_restart_state_reset(self):
 
         self.render_config_template(
             path=os.path.abspath(self.working_dir) + "/log/*",
-            clean_inactive="10s",
-            ignore_older="5s"
+            clean_inactive="20s",
+            ignore_older="10s"
```

Comment:
If we're waiting for the FQDN to be resolved we don't need to change the input settings.

I can see `ignore_older` needing to be increased if it takes longer for Filebeat to start up and scan for the files, but I do not get the change in `clean_inactive`

Reply:
Oh! I think I this was added by accident.
You're right, we only need to change `ignore_older`